### DISCUSSION TITLE
Ntfy.sh notification support.

### DISCUSSION THREAD

**Discussion by Andre601** [upptime/upptime#862] [Ideas]
Would be cool if support for [ntfy.sh](https://ntfy.sh) could be added.
This would allow us to receive push notifications on our phone.

Integration should be doable, even through GitHub Actions itself!

**Comment by shimaore** [comment]
I suspect the code addition should happen in https://github.com/upptime/uptime-monitor/blob/master/src/helpers/notifme.ts

**Comment by W-i-n-7** [comment]
> Would be cool if support for [ntfy.sh](https://ntfy.sh) could be added. This would allow us to receive push notifications on our phone.
> 
> Integration should be doable, even through GitHub Actions itself!

same idea
seems like after 1,5 years it is not getting added :(

**Reply by Andre601** [reply]
The main issue is, that the repo owner simply lacks time, as they have stated before.
It would generally be easier to submit a PR to implement this.
